### ISSUE TITLE
Bind template

### ISSUE THREAD

**Issue by 5tormChild** [fttx/barcode-to-pc-app#308]
We would need to bind a certain phone to a certain template on the server, selecting the template before each reading is not an option.
Because we're using 40+ phones simultaneously for barcode reading, separating the different purposes would be very important for us.
There should be a default template on the server, so if there is no forced template on the phone, it would use the default, or when the forced template is missing, it could fall back to the default one.

This is something we just can't live without, and in the current state B2PC doesn't support forcing / binding server and template.

**Comment by fttx** [comment]
Are you looking for an option to set a default template from the server settings? Currently, it seems that the app always loads the first template on the list or the most recently selected output template. Could you confirm if you are using it with multiple servers?

Thanks

**Comment by 5tormChild** [comment]
No, I need to force a specific template in the Android client, using the
first or selecting it before each reading is not an option.
Why can we have multiple templates, if the client loads the first, when not
selecting manually each time?
As I wrote, having a DEFAULT (fallback) template on the server, and letting
us bind a specific client to a specific template would make B2PC a lot more
versatile!
In our case we have 40+ phones using the one and only template, BECAUSE
modifying this template and making a mistake could render a few hours of
readings unusable or missing.

So on the server side make an option to select a default template, and on
the client side let's force a specific template, and when it's deleted on
the server, just fall back to the default one.

**Comment by 5tormChild** [comment]
"Could you confirm if you are using it with multiple servers?" - no, we're
using only one server with the Unlimited license, but we have 40+ phones
using your Android client.

What we need is, let's think together, let's assume we have templates T1
(this could be the default, but it should be adjustable), T2, T3, T4, T5.
We have specific readers that need to bind with T3, several others that
need to bind with T5, and on those phones where there's no binding, they
should use the default (in our case the T1) template.

So it wouldn't mess up your current workflow, because when there's no
forced binding set, it should work as of now.

filippo ***@***.***> ezt írta (időpont: 2023. jún. 29., Cs,
9:21):
